packages/core/src/bags/redux/selectors.js

```diff
@@ -465,3 +467,83 @@ export const getItemWholeQuantity = (state, bagItem) => {
 
   return quantity || 0;
 };
+
+/**
+ * Retrieves a specific bag operation by its id,.
+ *
+ * @function
+ *
+ * @param {object} state - Application state.
+ * @param {string} bagOperationId - Unique identifier of the operation.
+ *
+ * @returns {object} - Bag operation entity for the given id.
+ *
+ * @example
+ * import { getBagOperation } from '@farfetch/blackout-core/bags/redux';
+ *
+ * const mapStateToProps = (state, { bagOperation: { id } }) => ({
+ *     bagOperation: getBagOperation(state, id)
+ * });
+ */
+
```

Comment:
I think this empty line is not supposed to exist for the JsDoc 🤔

Reply:
Removed!